lib/omniauth/login_dot_gov/version.rb

```diff
@@ -1,5 +1,5 @@
 module OmniAuth
   module LoginDotGov
-    VERSION = '0.1.2'.freeze
```

Comment:
Should this be bumped to a new major version (1.0.0), since updating the gem would "break" the logout functionality for folks still passing `id_token`?

Reply:
Version is now 1.0.0